apps/ledger-live-desktop/src/renderer/components/OnboardingAppInstall/index.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { Flex, Text, Button } from "@ledgerhq/react-ui";
+import { Device } from "@ledgerhq/live-common/hw/actions/types";
+import { DeviceModelInfo } from "@ledgerhq/types-live";
+import { useTranslation } from "react-i18next";
+import { getDeviceModel } from "@ledgerhq/devices";
+import useFeature from "@ledgerhq/live-common/featureFlags/useFeature";
+
+import DefaultAppsIllustration from "./DefaultAppsIllustration";
+import RestoreAppsIllustration from "./RestoreAppsIllustration";
+import CancelModal from "./CancelModal";
+import InstallSetOfApps from "./InstallSetOfApps";
+
+const fallbackDefaultAppsToInstall = ["Bitcoin", "Ethereum", "Polygon"];
+
+type Props = {
+  device?: Device | null | undefined;
+  restoreDevice?: DeviceModelInfo | null | undefined;
+  onComplete?: () => void;
```

Comment:
IMO this prop shouldn't be optional (your use cases actually both provide this function). Then you can remove  your `handleComplete` function.

Reply:
Done